### ISSUE TITLE
Feature request: delete tab directly from bin

### ISSUE THREAD

**Issue by AntonioGHub** [qzb/dustman#33]
Nice to have: the ability to definitively remove a tab from the bin list (for instance a "X" button along each tab in the list).
This way one can clean the trash bin and focus on restoring the tabs she needs.

**Comment by mbid** [comment]
I agree, that would be useful. I'll implemented it when I come around to it.

**Comment by AntonioGHub** [comment]
thanks.
And, great addon by the way...

**Comment by sergeevabc** [comment]
I am shocked there is no way to remove tab from Dustbin: either within popup or by invoking tab per se.

**Comment by sergeevabc** [comment]
Alright, I found a replacement called [TabWrangler][1], which does the job.

[1]: https://addons.mozilla.org/en-US/firefox/addon/tabwrangler/
